src/components/remote_control/src/resource_allocation_manager_impl.cc

```diff
@@ -185,23 +262,25 @@ void ResourceAllocationManagerImpl::OnDriverDisallowed(
   list_of_rejected_resources.push_back(module_type);
 }
 
-void ResourceAllocationManagerImpl::OnUnregisterApplication(
-    const uint32_t app_id) {
+void ResourceAllocationManagerImpl::OnSDLEvent(
```

Comment:
@dev-gh Please rename t OnApplicaitonEvent

Reply:
@LuxoftAKutsan 
Now we have 4 cases: app exit, app unregistration, mobile apps disabling and PTU and I don't think all these are "application" event but some of them system (SDL) event. I think "SDL events" more general as it is related to both application (exit, unregistering) and external (mobile apps functionality disabled, PTU revoked some modules). Also last two events are handled w/o providing exact application id.

Reply:
@dev-gh each of your enum values contains Application word. You still don't think that it is application event?

Reply:
@LuxoftAKutsan 
As I said before there are two "external" events kApplicationPolicyUpdated and kApplicationsDisabled. In both these cases there is no specific application id passed.
I propose to rename kApplicationPolicyUpdated to kPolicyUpdated to highlight that and kApplicationsDisabled already mentions "applicationS" so I consider it as "common for all applications". Because of that I still don't think renaming is good here.

Reply:
@dev-gh I rely don't like that application_id is default argument, and we have couple of events that do not incudes appliaction id, What if in future we will need some additional arguments for SDL events? 
may be some boolean value, or policy table, or struct or list of applications? or device handle? So I prefer couple of interfaces OnApplicationEvent, OnPolicyEvent, OnDevice event and so on. It would be more more expandable. For example we will need to handle device disconnect.

Reply:
@LuxoftAKutsan 
Addressed with #256